pkg/config/validation/validation.go

```diff
@@ -3188,21 +3189,29 @@ var ValidateWorkloadEntry = registerValidateFunc("ValidateWorkloadEntry",
 	})
 
 func validateWorkloadEntry(we *networking.WorkloadEntry) (Warning, error) {
+	errs := Validation{}
+
 	addr := we.Address
 	if addr == "" {
-		return nil, fmt.Errorf("address must be set")
+		nw := we.Labels[label.TopologyNetwork.Name]
```

Comment:
We did not say TopologyNetwork can be applied to wle, i think we should not expand it since we have network under spec